learn/getting_started/supported_os.md

```diff
@@ -0,0 +1,21 @@
+# Supported operating systems
+
+Please ensure your operating system is listed on this page before installing the Meilisearch binary. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.
+
+If the provided binaries don't work for you, we recommend compiling Meilisearch from the source files. If the compilation fails, Meilisearch is not compatible with your machine.
+
+::: warning
+Any operating system or version not listed on this page does not mean Meilisearch won't work with it at all, we just don't guarantee it.
+:::
+
+## Linux
+
+The Meilisearch binary works on all Linux distributions with `amd64/x86_64` or `aarch64/arm64` architecture using glibc 2.27 and later.
```

Comment:
Maybe give the command to know the glibs version (I put in the engine team issue)?